probe-rs/src/debug/variable.rs

```diff
@@ -178,13 +178,12 @@ pub enum VariableType {
     Enum(String),
     /// Namespace refers to the path that qualifies a variable. e.g. "std::string" is the namespace for the strucct "String"
     Namespace,
-    /// A Pointer is a variable that contains a reference to another variable
+    /// A Pointer is a variable that contains a reference to another variable, and the type of the referenced variable may not be known until the reference has been resolved.
     Pointer(Option<String>),
     /// A Rust array.
     Array {
-        // TODO: Use a proper type here, not variable name
-        /// The name of the variable.
-        entry_type: VariableName,
+        /// The type name of the variable.
+        entry_type: String,
```

Comment:
```suggestion
        entry_type: String,
```

can we call this something along the lines `item_type_name`? I think that makes more sense to most people :)

Reply:
Done :)